src/main/java/game/racinggame/console/RacingConsole.java

```diff
@@ -0,0 +1,90 @@
+package game.racinggame.console;
+
+import game.InputDevice;
+import game.OutputDevice;
+import game.racinggame.RacingCar;
+import game.racinggame.RacingDataTransferObject;
+import game.racinggame.RacingRankingMachine;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static game.constant.NumberConstant.ZERO;
+
+final public class RacingConsole implements InputDevice, OutputDevice {
+
+    private static final String COMMA = ",";
+    private List<String> carNames = null;
+    private int tryCount = 0;
+
+    public RacingConsole(){}
+
+    @Override
+    public Object doInput() {
+
+        System.out.println("경주할 자동차 이름을 입력하세요. (이름은 쉼표(,) 를 기준으로 구분)");
+        final String carNames = scanner.nextLine();
+
+        System.out.println("시도할 회수는 몇 회 인가요?");
+        final int tryCount = toInt(scanner.nextLine());
+
+        final List<String> carList = splitByComma(carNames);
+
+        validateCarsCheck(carList);
+        validateInputCheck(tryCount);
+
+        return new RacingDataTransferObject(splitByComma(carNames), tryCount);
```

Comment:
```suggestion
        return new RacingDataTransferObject(carList, tryCount);
```

carList가 들어가는 게 맞지않을까용?

Reply:
저도 같은 의견입니다! 검증까지 끝난? carList를 넣는게 나을 것 같아요

Reply:
아하 넵넵. 검증을 끝내고 난 뒤의 데이터를 넣는 것이 좋겠군요. ! 감사합니다